Dockerfile

```
@@ -1,11 +1,9 @@
 # Elixir and phoenix assets build image
-FROM elixir:1.8.2-otp-22 as code_builder
+FROM elixir:1.8.2-otp-22-alpine as code_builder
 
 ENV MIX_ENV prod
 
-RUN apt-get update \
-  && curl -sL https://deb.nodesource.com/setup_8.x | bash \
-  && apt-get install -y nodejs git
+RUN apk add --update nodejs git make g++
```

I think using `--no-cache` is a better option here: https://github.com/gliderlabs/docker-alpine/blob/master/docs/usage.md#disabling-cache

```suggestion
RUN apk add --no-cache nodejs git make g++
```

---

Should we do this in all places?